includes/admin/class-wc-payments-admin.php

```diff
@@ -166,6 +166,18 @@ public function add_payments_menu_for_treatment() {
 				],
 			]
 		);
+		wc_admin_register_page(
```

Comment:
Remove this later as we don't want the Onboarding to appear in the WC Admin menu.